src/main/java/com/aserto/config/Security.java

```diff
@@ -18,9 +23,28 @@ public Security(AuthzConfig authzCfg) {
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http.cors(Customizer.withDefaults())
-            .csrf(AbstractHttpConfigurer::disable).authorizeHttpRequests(authorize -> authorize
-                .anyRequest().access(new AsertoAuthorizationManager(authzCfg))
-        );
+            .csrf(AbstractHttpConfigurer::disable)
+            .authorizeHttpRequests(authorize -> authorize
+                .requestMatchers(HttpMethod.DELETE, "/todos/{id}")
+                .access(new AsertoAuthorizationManager(authzCfg))
+
+                .requestMatchers(HttpMethod.GET, "/todos")
+                .access(new AsertoAuthorizationManager(authzCfg))
+
+                .requestMatchers(HttpMethod.GET, "/users/{userID}")
+                .access(new AsertoAuthorizationManager(authzCfg))
+
+                // the post authorization can be written here or as an annotation in the controller
+                // if we want to have the condition in the controller we permit the traffic to pass here and block it in the controller
+                .requestMatchers(HttpMethod.POST, "/todos")
+                .permitAll()
+//                .access(new CheckConfig(authzCfg, "resource-creator", "resource-creators", "member").getAuthManager())
```

Comment:
This should be uncommented

Reply:
There are 2 ways the check middleware can be used. The commented code is an example, another way is to do the authentication at the method level in the controller.
e.g.
```java
@PreAuthorize("@check.objectType('resource-creator').objectId('resource-creators').relation('member').allowed()")
```
The second way of doing authentication is used for the POST method

Reply:
I tend to agree with @oanatmaria. This commented out code is confusing.
It makes sense to describe the two different ways to configure authorization for a route in the java SDK documentation, but here in the sample app I think we should just pick one and use it.

I would be in favor of consistency. If we configure authorization for all other routes here in this function, why not configure the POST too?

Reply:
I changed the POST to use path-level authorization instead of method authorization to make it similar to the other requests.